### ISSUE TITLE
LDAP push notification: can not find user by username

### ISSUE THREAD

**Issue by wangchunyang** [SUSE/Portus#430]
If LDAP is enabled, a push notification is handled by Portus and got error as follow:
```
  User Load (1.0ms)  SELECT  `users`.* FROM `users` WHERE `users`.`username` = 'chun-yang.wang@hpe.com' LIMIT 1
  Cannot find user chun-yang.wang@hpe.com
```

To fix it, the code should be changed to use "ldap_name" to find user instead of "username" field.

**Comment by wangchunyang** [comment]
A pull request with the following code was created.

```ruby
  # Find the user that can be guessed from the given push event.
  def self.find_from_event(event)
    if Portus::LDAP.enabled?
      actor = User.find_by(ldap_name: event["actor"]["name"])
    else
      actor = User.find_by(username: event["actor"]["name"])
    end
    logger.error "Cannot find user #{event["actor"]["name"]}" if actor.nil?
    actor
  end
```

**Comment by mssola** [comment]
Fixed with PR #431.
